<ui version="4.0" >
 <class>PrefInput</class>
 <widget class="QWidget" name="PrefInput" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="input_tab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Keyboard</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="keyboard_icon" >
           <property name="text" >
            <string>icon</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="actioneditor_desc" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Here you can change any key shortcut. To do it double click or press enter over a shortcut cell. Optionally you can also save the list to share it with other people or load it in another computer.</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="ActionsEditor" native="1" name="actions_editor" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>&amp;Mouse</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item rowspan="4" row="0" column="0" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="mouse_icon" >
           <property name="text" >
            <string>icon</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1_16" >
           <property name="text" >
            <string>Button functions:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="0" column="0" >
            <widget class="QLabel" name="textLabel2_6" >
             <property name="text" >
              <string>&amp;Left click</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
             <property name="buddy" >
              <cstring>left_click_combo</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="4" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>81</width>
               <height>31</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="3" colspan="2" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>111</width>
               <height>31</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" colspan="2" >
            <widget class="QLabel" name="textLabel3_4" >
             <property name="text" >
              <string>&amp;Double click</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
             <property name="buddy" >
              <cstring>double_click_combo</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="MyComboBox" name="double_click_combo" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>7</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3" >
            <widget class="MyComboBox" name="left_click_combo" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>7</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="3" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <widget class="Line" name="line7" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" colspan="2" >
          <widget class="QLabel" name="textLabel2_8" >
           <property name="text" >
            <string>&amp;Wheel function:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>wheel_function_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QComboBox" name="wheel_function_combo" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text" >
             <string>Media seeking</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Volume control</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Zoom video</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>121</width>
             <height>31</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionsEditor</class>
   <extends>QWidget</extends>
   <header>actionseditor.h</header>
  </customwidget>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
